modules/cloudtrail/variables.tf

```diff
@@ -117,3 +117,39 @@ variable "cloudtrail_assume_role_policy_document" {
    }
   EOF
 }
+
+variable "log_metrics" {
+  type = object({
+    enabled           = optional(bool, false)
+    metrics_namespace = optional(string, "LogBasedMetrics")
+    enabled_metrics   = optional(list(string), []) # Possible values are: iam-user, iam-role, iam-policy, s3, root-account, elastic-ip, elastic-network, rds, cdn, lambda, elasticache, sns, sqs, ec2, elasticsearch, elb.
+    alerts = optional(object({
+      enabled = optional(bool, false)
+    }))
+  })
+  default     = { enabled : false }
+  description = "Provide CloudWatch Log Metric filters"
+}
+
+variable "alarm_actions" {
```

Comment:
should be no need for this variable and the only topic_name will go into above variable as sub-field